pegasus/sites.v3/hourofcode.com/views/home_highlights.haml

```diff
@@ -3,11 +3,9 @@
     .col-md-8
       %br/
       %h2
-        Hour of Code
-        = campaign_date('year').to_i - 1
+        Hour of Code Highlights
       %a.twitter-grid{href: "https://twitter.com/codeorg/timelines/925449654308704256?limit=4?ref_src=twsrc%5Etfw?limit=4"}
-        The Hour of Code
-        = campaign_date('year').to_i - 1
+        Hour of Code Highlights
```

Comment:
This Twitter feed only appears in English so need to localize.

I'm not sure where this second "The Hour of Code 2017" string appears. I don't see it on the page, but it does appear in the page source. Perhaps it's what shows up when the twitter feed doesn't load properly? So I decided to update it as well.

Reply:
I believe you're right.  The page source has an `<a class="twitter-grid" ...` which gets replaced by an `<iframe ...` when this grid is loaded by JS.